docs/usage.md

```diff
@@ -30,26 +31,28 @@ command.
 
 ## version
 
-The version command provides version information, returning either the semver version by default or the git sha when passed `--sha`.
+The version command provides version information, returning either the [semver version](https://semver.org/) by default, the git commit sha when passed `--sha`, or verbose output when passed `--full`.
```

Comment:
FYI. Will change semver version to just version as openapi-generator versioning doesn't strictly follow semver